regions/shapes/ellipse.py

```diff
@@ -102,16 +102,27 @@ def to_mask(self, mode='center', subpixels=5):
         else:
             use_exact = 1
 
-        fraction = elliptical_overlap_grid(xmin, xmax, ymin, ymax, nx, ny,
-                                           self.major, self.minor,
-                                           self.angle.to(u.deg).value,
-                                           use_exact, subpixels)
+        fraction = elliptical_overlap_grid(
+            xmin, xmax, ymin, ymax, nx, ny,
+            self.major, self.minor,
+            self.angle.to(u.deg).value,
+            use_exact, subpixels,
+        )
 
         return Mask(fraction, bbox=bbox)
 
     def as_patch(self, **kwargs):
-        # TODO: needs to be implemented
-        raise NotImplementedError
+        """Matplotlib patch object for this region (`matplotlib.patches.Ellipse`).
+        """
+        from matplotlib.patches import Ellipse
+        xy = self.center.x, self.center.y
+        width = 2 * self.major
+        height = 2 * self.minor
+        # TODO: think about how to map major / minor / angle to MPL parameters.
```

Comment:
I don't see any changes to rectangle.  `photutils.RectangularAperture` is correct as defined, where the `width` side is along the `x` axis for `theta=0` and `theta` increases counterclockwise.

Note that the all of these plots require the "normal" display convention where the `(0, 0)` pixel is at the lower left (i.e. `origin='lower'`).  Otherwise the rotation angle will be in the opposite direction.